tests/staking/StakingTest.js

```diff
@@ -731,6 +724,302 @@ contract("Staking", (accounts) => {
         });
     });
 
+    describe("extendStakingDuration", () => {
+        // it("should fail if second stake in the same block", async () => {
```

Comment:
LGTM, but are we good to remove this test case?

Reply:
Sorry, haven't pushed last commit with uncommented test.